src/main/Dashboard.java

```diff
@@ -1,10 +1,131 @@
 package main;
 
+import java.util.HashSet;
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
 public class Dashboard {
 
-	public static void main(String[] args) {
-		// TODO Auto-generated method stub
+    static HashSet<User> hashUsers = new HashSet<User>();
```

Comment:
Avoid package-level visibility, it's rarely needed.
Encapsulation is key in object-oriented programming, so keep everything as self-contained as possible and only expose fields and methods to the public when it's necessary.

When in doubt, make things `private`.